intent_brokering/README.md

```diff
@@ -0,0 +1,221 @@
+# Service Discovery
```

Comment:
Should this be titled "Intent Brokering"?

Reply:
Good catch, thanks!